ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/management-api-http.yml

```diff
@@ -1,4 +1,4 @@
-# Management API HTTP
+---
```

Comment:
```suggestion
---
# Management API HTTP
```

Reply:
I removed these comments as this file contains only one parent key.

Reply:
I don't think we need the comments since they are just repeating the filename and single key.